README.md

```diff
@@ -98,6 +106,33 @@ The `decimal.Decimal` is, however, a practical choice for use cases where float
 Illegal types are `dict` and `list`, and their subtypes.
 A `ValueError` will be raised if `parse_float` produces illegal types.
 
+### Building a `tomli`/`tomllib` compatibility layer<a name="building-a-tomlitomllib-compatibility-layer"></a>
+
+Python versions 3.11+ ship with a version of Tomli:
+the `tomllib` standard library module.
+To build code that uses the standard library if available,
+but still works seamlessly with Python 3.6+,
+do the following.
+
+Instead of a hard Tomli dependency, use the following
+[PEP 508](https://www.python.org/dev/peps/pep-0508/) string
```

Comment:
IMO it's better to link here, rather than directly to the PEP:
```suggestion
[dependency specifier](https://packaging.python.org/en/latest/specifications/dependency-specifiers/)
```